components/cards.css

```diff
@@ -0,0 +1,135 @@
+.card {
+  max-width: 18.75rem;
+  background-color: #171717;
+  border-radius: 5px;
+  text-align: center;
+
+  position: relative;
+}
+
+.card__image {
+  border-radius: 5px;
+}
+
+.card__title {
+  font-size: 1rem;
+  line-height: 1.5rem;
+  margin: 0.5rem 0 0.5rem;
+}
+
+.card__subtitle {
+  font-size: 1.5rem;
+  font-weight: 700;
+  line-height: 2.25rem;
+  margin-bottom: 1rem;
+}
+
+.card__subtitle--canceled {
+  font-weight: 400;
+  font-size: 1.5rem;
+  line-height: 2.25rem;
+  text-decoration: line-through;
+}
+
+.card__content > .btn {
+  width: 100%;
+  padding: 0.5rem 0;
+  margin: 0;
+}
+
+.card__badge {
+  background: var(--blue);
+  color: var(--white);
+  font-weight: 500;
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  padding: 0.25rem 0.75rem;
+  position: absolute;
+  top: 1rem;
+}
+
+.card__dismiss {
+  cursor: pointer;
+  padding: 0.7rem 1rem;
+  position: absolute;
+  right: 0;
+}
+
+.card__overlay {
+  color: var(--white);
+  background-color: rgba(0, 0, 0, 0.5);
+  font-size: 1.5rem;
+
+  position: absolute;
+  inset: 0;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/********* horizontal card *********/
+.card--horizontal {
+  width: 43.75rem;
+  max-width: 100%;
+  display: flex;
+  text-align: left;
+}
+
+.card--horizontal .card__image {
+  max-height: 24.375rem;
+}
+
+.card--horizontal .card__content {
+  margin: 2.5rem;
+  flex-grow: 1;
+
```

Comment:
also here

Reply:
this class is a flex children as well as flex container, so i've separated the flex children and flex container properties with an empty line

Reply:
appreciate for your review